services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -156,19 +162,20 @@ public RouterHttpRequestStats(
           new Max(),
           TehutiUtils.getFineGrainedPercentileStat(getName(), getFullMetricName(keySizeSensorName)));
 
-      responseSizeSensor = registerSensor(
-          responseSizeSensorName,
+      String valueSizeSensorName = "value_size_in_byte";
+      valueSizeSensor = registerSensor(
```

Comment:
This looks weird..

Can we get rid of the new sensor but keep using the existing `responseSizeSensor`?
Essentially, when `isKeyValueProfilingEnabled` is `true`, we only use `FineGrainedPercentileStat` for total stats.
It can be done like this:

```
if (isKeyValueProfilingEnabled && storeName.equals(AbstractVeniceAggStats.STORE_NAME_FOR_TOTAL_STAT)) {
      String keySizeSensorName = "key_size_in_byte";
      keySizeSensor = registerSensor(
          keySizeSensorName,
          new Avg(),
          new Max(),
          TehutiUtils.getFineGrainedPercentileStat(getName(), getFullMetricName(keySizeSensorName)));

      responseSizeSensor = registerSensor(
          responseSizeSensorName,
          new Avg(),
          new Max(),
          TehutiUtils.getFineGrainedPercentileStat(getName(), getFullMetricName(responseSizeSensorName)));
    } else {
      responseSizeSensor = registerSensor(
          responseSizeSensorName,
          new Avg(),
          new Max(),
          TehutiUtils.getPercentileStat(getName(), getFullMetricName(responseSizeSensorName)));
    }

```